Formula/libxcb.rb

```diff
@@ -14,10 +14,10 @@ class Libxcb < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "993d37bb436fab0157ed5f3c031f9a18168053439e93edb4bdce9abe6e99373d"
   end
 
+  depends_on "libpthread-stubs" => :build
```

Comment:
Note that this means `pkg-config --cflags xcb` won't work unless user manually installs `libpthread-stubs`:
```console
% pkg-config --cflags xcb
Package pthread-stubs was not found in the pkg-config search path.
Perhaps you should add the directory containing `pthread-stubs.pc'
to the PKG_CONFIG_PATH environment variable
Package 'pthread-stubs', required by 'xcb', not found
```

Due to `lib/pkgconfig/xcb.pc`:
```
Requires.private: xau >= 0.99.2 pthread-stubs xdmcp
```

---

On side note, should be macOS-only:
```
Release 1.14 (2020-02-22)
=========================
..
* Only require pthread-stubs on non-Linux platforms
```

Reply:
I opened a [merge request](https://gitlab.freedesktop.org/xorg/lib/libxcb/-/merge_requests/34), waiting for feedback now.